src/main/java/eu/dissco/core/digitalspecimenprocessor/repository/DigitalSpecimenRepository.java

```diff
@@ -77,6 +75,31 @@ private Query specimenToQuery(DigitalSpecimenRecord digitalSpecimenRecord) {
         .set(NEW_DIGITAL_SPECIMEN.SOURCE_SYSTEM_ID,
             digitalSpecimenRecord.digitalSpecimen().attributes().get("ods:sourceSystemId").asText())
         .set(NEW_DIGITAL_SPECIMEN.CREATED, digitalSpecimenRecord.created())
+        .set(NEW_DIGITAL_SPECIMEN.LAST_CHECKED, Instant.now()).set(NEW_DIGITAL_SPECIMEN.DATA,
```

Comment:
Hi Sou, I see what you mean and normally I would fully agree. Here it is however slightly different. This is an upsert statement. What that means is that we have an insert statement with a onConflict (in this case if the ID is already there) and then the actions on what to update (everything outside of the ID can be updated). The Instant.now() will therefore never be called twice. It will either use the insert or will use the update. 
This enables us to do the insert/update in one statement/call instead of having to check and then either updating or inserting